src/base/ToCpp.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef BASE_TOCPP_H
+#define BASE_TOCPP_H
+
+#ifdef __cplusplus
+
+// function. The C++ function suffixed with the _cpp extension
```

Comment:
The first description line was dropped during the move. I am polishing the description slightly in this suggestion, but I do not insist on any polishing, just on restoring the missing line.

```suggestion
/// Defines a C++ equivalent of an extern "C" function.
/// The defined C++ function name uses a _cpp suffix.
```